opa/Dockerfile

```
@@ -20,6 +38,7 @@ RUN microdnf update && \
     microdnf clean all
 
 COPY opa/licenses /licenses
+COPY --from=opa-bundle-builder /opa-bundle-builder/target/release/stackable-opa-bundle-builder /stackable/opa-bundle-builder
```

Nit move down below `RUN curl -L https://repo.stackable.tech/repository/packages/opa/opa_${PRODUCT}.tar.gz` and use
```suggestion
COPY --from=opa-bundle-builder --chown=stackable:stackable /opa-bundle-builder/target/release/stackable-opa-bundle-builder /stackable/opa-bundle-builder
```